renku/cli/update.py

```diff
@@ -79,6 +79,42 @@
 To avoid excesive recreation of the large portion of files which could have
 been affected by a simple change of an input file, consider speficing a single
 file (e.g. ``renku update G``). See also :ref:`cli-status`.
+
+.. _cli-update-with-siblings:
+
+Update siblings
+~~~~~~~~~~~~~~~
+
+If a tool produces multiple output files, these outputs need to be always
+updated together.
+
+.. code-block:: text
+
+                   (B)
+                  /
+    *A*--[step 1]--(C)
+                  \
+                   (D)
+
+Update of a single file would end up with following error.
```

Comment:
Clearer wording:

    An attempt to update a single file would fail with the following error.